data/xml/2014.amta.xml

```diff
@@ -0,0 +1,677 @@
+    </paper>
+    <paper id="27">
+      <title>A tunable language model for statistical machine translation</title>
+      <author><first>Junfei</first><last>Guo</last></author>
+      <author><first>Juan</first><last>Liu</last></author>
+      <author><first>Qi</first><last>Han</last></author>
+      <author><first>Andreas</first><last>Maletti</last></author>
+      <pages>356-368</pages>
+      <url hash="ced80d07">2014.amta-researchers.27</url>
+      <abstract>A novel variation of modified KNESER-NEY model using monomial discounting is presented and integrated into the MOSES statistical machine translation toolkit. The language model is trained on a large training set as usual, but its new discount parameters are tuned to the small development set. An in-domain and cross-domain evaluation of the language model is performed based on perplexity, in which sizable improvements are obtained. Additionally, the performance of the language model is also evaluated in several major machine translation tasks including Chinese-to-English. In those tests, the test data is from a (slightly) different domain than the training data. The experimental results indicate that the new model significantly outperforms a baseline model using SRILM in those domain adaptation scenarios. The new language model is thus ideally suited for domain adaptation without sacrificing performance on in-domain experiments.</abstract>
+    </paper>
+  </volume>
+  <volume id="users" ingest-date="2021-06-20">
+    <meta>
+      <booktitle>Proceedings of the 11th Conference of the Association for Machine Translation in the Americas: MT Users Track</booktitle>
+      <publisher>Association for Machine Translation in the Americas</publisher>
+      <address>Vancouver, Canada</address>
+      <month>October 22-26</month>
+      <year>2014</year>
+      <editor><first>Yaser</first><last>Al-Onaizan</last></editor>
```

Comment:
I think I would just list "Olga Beregovaya, Mike Dillinger, Jennifer Doyon, Raymond Flournoy, Patti O�Neill-Brown & Chuck Simmons" as the editors" for this volume (per the web page).